be/src/vec/aggregate_functions/aggregate_function.h

```diff
@@ -43,7 +43,7 @@ class IDataType;
 
 struct AggregateFunctionAttr {
     bool enable_decimal256 {false};
-    std::vector<std::pair<std::string, bool>> column_infos;
+    std::vector<std::string> column_names;
```

Comment:
Why change this basic data structure?

Reply:
Optimize the redundant data structures added in approx_top_k.